wrangler-transform/src/e2e-test/resources/BQtesdata/BigQuery/BigQueryInsertDataQuery.txt

```diff
@@ -0,0 +1,9 @@
+insert into `DATASET.TABLE_NAME` (ID, name, email, Customer_id, First_name, Last_name, Age, Address, Col1,
```

Comment:
Since the "body" column is used in the second test (parse csv, fill null, send to error) update the data to reflect an actual csv string. 

Also, update the address column to have one null value to ensure the send-to-error if empty directive that is used is working correctly.

Reply:
Done.